ads/inabox/inabox-messaging-host.js

```diff
@@ -62,16 +62,37 @@ class NamedObservable {
   }
 }
 
+/** @typedef {{
+      iframe: !HTMLIFrameElement,
+      measurableFrame: !HTMLIFrameElement,
+      observeUnregisterFn: (!UnlistenDef|undefined),
+  }} */
+let AdFrameDef;
```

Comment:
can this be a const?

Reply:
typedefs are declared as let (they don't actually hold state)